java/IBAutomater/src/ibautomater/Common.java

```diff
@@ -103,16 +103,21 @@ public static JTextField getTextField(Container container, int index) {
     }
 
     public static JMenuItem getMenuItem(Container container, String menuText, String menuItemText) {
-        JMenuBar menuBar = ((JFrame)container).getJMenuBar();
-        for (int i = 0; i < menuBar.getMenuCount(); ++i) {
-            JMenu menu = menuBar.getMenu(i);
-            if (!menu.getText().equals(menuText)) continue;
-            for (int j = 0; j < menu.getItemCount(); ++j) {
-                JMenuItem menuItem = menu.getItem(j);
-                if (menuItem == null || !menuItem.getText().equalsIgnoreCase(menuItemText)) continue;
-                return menuItem;
+        try {
+            JMenuBar menuBar = ((JFrame) container).getJMenuBar();
+            for (int i = 0; i < menuBar.getMenuCount(); ++i) {
+                JMenu menu = menuBar.getMenu(i);
+                if (!menu.getText().equals(menuText)) continue;
+                for (int j = 0; j < menu.getItemCount(); ++j) {
+                    JMenuItem menuItem = menu.getItem(j);
+                    if (menuItem == null || !menuItem.getText().equalsIgnoreCase(menuItemText)) continue;
+                    return menuItem;
+                }
             }
         }
+        catch (Exception e) {
+            return null;
```

Comment:
Maybe better add the missing null checks (`container` and `menuBar`), instead of using a `try...catch` block which would hide other errors?

Reply:
done!